modules/azure-oracle-adbs/README.md

```diff
@@ -0,0 +1,57 @@
+# azure-oracle-adbs
+
+## Summary
+
+Terraform module for **Oracle Autonomous Database Service (ADBS)** , Creates a new Autonomous Database resource in Oracle Database @ Azure.
```

Comment:
It's should be "[Oracle Autonomous Database Serverless (ADB-S)](https://docs.oracle.com/en-us/iaas/Content/multicloud/oaa.htm#oaasystems)".